### ISSUE TITLE
Replacement rule for .NET?

### ISSUE THREAD

**Issue by chalin** [sapegin/textlint-rule-terminology#64]
The following replacement rules don't catch ".net". Any idea why? Which rules might work if these aren't good?

```json
  "rules": {
    "terminology": {
      "defaultTerms": false,
      "skip": [],
      "terms": [
        ["\\.net", ".NET"],
        ["(?<=\\.)net\\b", "NET"],
```
